diesel_compile_tests/tests/fail/derive/bad_not_sized.rs

```diff
@@ -0,0 +1,9 @@
+extern crate diesel;
+use diesel::expression::AsExpression;
+
+#[derive(AsExpression)]
+#[diesel(sql_type = bool)]
```

Comment:
Minor nit:
```suggestion
```

(Mind that just applying this via Github's UI will not work, as we need to update the output as well)